components/application-operator/charts/application/templates/authorization-policy.yaml

```diff
@@ -0,0 +1,121 @@
+{{- if eq .Values.global.strictMode "enabled" }}
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: {{ .Release.Name }}-connectivity-validator
+  labels:
+    app: {{ .Release.Name }}-connectivity-validator
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-connectivity-validator
+  rules:
+    - from:
+        - source:
+            principals:
+             - cluster.local/ns/istio-system/sa/istio-ingressgateway-service-account
+      to:
+        - operation:
+            methods:
+              - POST
+            paths:
+              - /{{ .Release.Name }}/v1/events
+              - /{{ .Release.Name }}/v2/events
+              - /{{ .Release.Name }}/events
+        - operation:
+            methods:
+              - GET
+              - PUT
+              - POST
+            paths:
+              - /{{ .Release.Name }}/v1/metadata*
+    - from:
+      - source:
+          principals:
+          - cluster.local/ns/kyma-system/sa/core-console-backend-service
+      to:
+      - operation:
+          methods:
+          - GET
+          paths:
+          - /v1/health
+---
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: {{ .Release.Name }}-event-service
+  labels:
+    app: {{ .Release.Name }}-event-service
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-event-service
+  rules:
+    - from:
+        - source:
+            principals:
+              - cluster.local/ns/kyma-integration/sa/{{ .Release.Name }}-connectivity-validator
+      to:
+        - operation:
+            methods:
+              - POST
+            paths:
+              - /{{ .Release.Name }}/v1/events
+              - /{{ .Release.Name }}/v2/events
+        - operation:
+            methods:
+              - GET
+            paths:
+              - /{{ .Release.Name }}/v1/events/subscribed
+              - /v1/health
+    - from:
+        - source:
+            principals:
+              - cluster.local/ns/kyma-system/sa/core-console-backend-service
```

Comment:
the 'core' comes from a different chart. would it make sense to parameterize it here as well instead of hardcoding?

Reply:
Parameterized the best I could but release `core` cannot be known to `application-operator` chart.